ajax/policyValue.php

```diff
@@ -48,7 +48,6 @@
 $policy = $factory->createFromDBByID($policyId);
 if ($policy !== null) {
    echo $policy->formGenerator($mode, [
-      'policyId' => $policyId,
```

Comment:
$policy contains an instance if the itemtype policy thanks to the id given to the factory. No need to pass it to the formGenerator method.